irohad/model/execution/impl/command_executor.cpp

```diff
@@ -73,13 +72,34 @@ namespace iroha {
     bool AppendRoleExecutor::hasPermissions(const Command &command,
                                             ametsuchi::WsvQuery &queries,
                                             const Account &creator) {
+      creator_ = creator;
```

Comment:
What's the point of copying?

Reply:
Object `creator` is located on the stack, I have to save it for future use